autoload/neomake/makers/ft/proto.vim

```diff
@@ -0,0 +1,10 @@
+function! neomake#makers#ft#proto#EnabledMakers() abort
+    return ['prototool']
+endfunction
+
+function! neomake#makers#ft#proto#prototool() abort
+    return {
+                \ 'exe': 'prototool',
+                \ 'args': ['lint'],
+                \ }
```

Comment:
I think there should be an errorformat, no?
Can you provide example output?

Reply:
I felt the output was useful by default:

```bash
λ prototool lint
example.proto:7:17:Expected field number.
```
<img width="717" alt="screen shot 2018-10-24 at 7 14 49 pm" src="https://user-images.githubusercontent.com/613740/47416310-27ba9780-d7c1-11e8-9392-52146cbafb70.png">

```bash
λ prototool lint
example.proto:1:1:File option "go_package" is required.
example.proto:1:1:File option "java_multiple_files" is required.
example.proto:1:1:File option "java_outer_classname" is required.
example.proto:1:1:File option "java_package" is required.
example.proto:1:1:No package declaration found.
```
<img width="716" alt="screen shot 2018-10-24 at 7 18 05 pm" src="https://user-images.githubusercontent.com/613740/47416464-8122c680-d7c1-11e8-9770-8a18b2cb434c.png">

Reply:
Makes sense! I've updated the PR to include the error format.

Yes I'm using `g:neomake_highlight_lines = 1`. A recent switch for me and I'm still undecided on whether I'll keep it on :)

Reply:
Re `g:neomake_highlight_lines = 1`: I would assume that it is more useful to see the exact column if provided.
Have not checked, but maybe it makes sense to have a mode where columns are always used (if given / not 1), and highlight the line only otherwise?

Reply:
Yes I think a mode like that could be handy. Best of both worlds.

Reply:
Cool, https://github.com/neomake/neomake/pull/2116.